### PR TITLE
PR preview: link to PR and build

### PR DESCRIPTION
**Pull request by stsewd**
Needs https://github.com/readthedocs/readthedocs.org/pull/7828

### CODE REVIEW

**Comment by stsewd** [comment]
Damn, this is still running on travis :disappointed:

File: tox.ini [hunk 1]
```diff
@@ -15,22 +15,24 @@ deps =
...
     sphinx18: Sphinx<1.9
-    sphinx21: Sphinx~=2.1.0
-    sphinx22: Sphinx~=2.2.0
-    sphinx23: Sphinx~=2.3.0
-    sphinx24: Sphinx~=2.4.0
-    sphinx30: Sphinx~=3.0.0
+    sphinx21: Sphinx<2.2
+    sphinx22: Sphinx<2.3
+    sphinx23: Sphinx<2.4
+    sphinx24: Sphinx<2.5
```
Comment (ericholscher) [inline tox.ini:21]: I think we can just test the latest version of each major release, no? This seems like a lot of overkill.

**Review by ericholscher — APPROVED** [review]
This looks good to me. We should do CI changes in a diff PR in the future so we don't block them on each other.

File: tox.ini (hunk 1 above)
Comment (stsewd) [reply tox.ini:21]: I think this goes back to what versions we officially support. This extension is installed by default, so I think we should be careful here and try to not break the builds on any version. But if we are good just supporting the latest version of each release, I'm ok with that.

Comment (ericholscher) [reply tox.ini:21]: Yea, I guess for this it's good to be extra careful 👍

**Pull request merged**